packages/helpers/classes/mail.d.ts

```diff
@@ -152,6 +152,7 @@ export interface MailData {
   substitutionWrappers?: string[],
   
   isMultiple?: boolean,
+  dynamicTemplateData?: { [key: string]: string },
```

Comment:
I think it might need to be `dynamicTemplateData?: { [key: string]: any }`

because you can pass objects like this:

```javascript
'dynamic_template_data': {
  'subject': 'Hi!',
  'name': 'Example User',
  'location': {
    'city': 'Birmingham',
    'country': 'United Kingdom'
  }
}
```